HunnitLog/HunnitLog/View/Home/ScoreRow.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  ScoreRow.swift
+//  HunnitLog
+//
+//  Created by 초이 on 2020/12/05.
+//
+
+import SwiftUI
+
+struct ScoreRow: View {
+    @State private var sliderValue: Double = 0
+    
+    let title: String
+    let achievement: Int
+    
+    private enum Constants {
+        static let horizontalPadding: CGFloat = 18
+        static let verticalPadding: CGFloat = 15
+        static let trailing: CGFloat = 10
+        static let circleDiameter: CGFloat = 47
+        static let titleFont: Font = .system(size: 17)
+        static let achievementFont: Font = .system(size: 13)
+        static let borderWidth: CGFloat = 0.3
+        static let borderRadius: CGFloat = 13
+        
+        static let sliderMinValue: CGFloat = -5
+        static let sliderMaxValue: CGFloat = 5
+        
+        static let badImageName: String = "imgTextboxBad"
+        static let notBadImageName: String = "imgTextboxNotbad"
+        static let sosoImageName: String = "imgTextboxSoso"
+        static let greatImageName: String = "imgTextboxGreat"
+        static let excellentImageName: String = "imgTextboxExcellent"
+    }
+    
+    
+    var body: some View {
+        HStack(spacing: 7) {
+            VStack {
+                HStack {
+                    Circle()
+                        .fill(CustomColor.yellow)
+                        .frame(width: Constants.circleDiameter, height: Constants.circleDiameter)
+                        .padding(.trailing, Constants.trailing)
+                    
+                    VStack(alignment: .leading) {
+                        Text(title)
+                            .font(Constants.titleFont)
+                        Text("달성률 \(achievement)%")
+                            .font(Constants.achievementFont)
+                            .foregroundColor(CustomColor.coolGray)
+                    }
+                }
+                
+                GeometryReader { geometry in
+                    VStack {
+                        HStack {
+                            Emotion(rawValue: makeScore(sliderValue))?
+                                .image
+                                .frame(width: CGFloat(geometry.size.width / 10 * CGFloat(5.9 + sliderValue)), height: 32, alignment: .bottomTrailing)
```

Comment:
이렇게 frame 대신 offset을 사용해보는 것도 괜찮을 것 같습니다!
```suggestion
let baseOffset = CGFloat(sliderValue)
        + geometry.size.width/2
        - 35 //이미지 width의 1/2
Emotion(rawValue: makeScore(sliderValue))?
    .image
    .offset(x: baseOffset
            + CGFloat(geometry.size.width / 10 * CGFloat(sliderValue))
    )
```

Reply:
`.offset` 사용, min max 값 설정하여 커밋하겠습니다!
현재 iPhone 11에서 정상작동하고, 좌표 디테일은 모든 뷰 다 짠 후에 잡도록 하겠습니다 ~!
지선배 여러운 선배...